yjit.c

```diff
@@ -1165,8 +1165,8 @@ rb_yjit_invalidate_all_method_lookup_assumptions(void)
 }
 
 // Number of object shapes, which might be useful for investigating YJIT exit reasons.
-static VALUE
-object_shape_count(rb_execution_context_t *ec, VALUE self)
+VALUE
+rb_object_shape_count(void)
```

Comment:
Does this seem like the right change? We used to use it as a Primitive, now we are just calling it from rust.

Reply:
Yes, it does.

I personally prefer having an `rb_yjit_` prefix for non-static C functions in yjit.c, but existing functions don't necessarily follow the convention and I'm not sure if other files like shape.c would use this name, so the current function name seems also fine.